src/main/resources/messages/messages.properties

```diff
@@ -490,12 +490,9 @@ error.no_files_found_for_pre_translate=Couldn't find any files to Pre-Translate
 error.project_is_incomplete=The current project is incomplete
 
 error.glossary.build_glossary=Failed to build the glossary
-error.glossary.not_found_by_id=Couldn't find glossary by the specified ID
```

Comment:
Maybe it would be better to keep this message (as well as the `error.tm.not_found_by_id`)? It's more user-friendly than the API error message

Reply:
actually error from sever is prettified and does not look too technical. Example:

<img width="448" alt="image" src="https://github.com/crowdin/crowdin-cli/assets/16465671/0e23c04d-baeb-421c-bad7-28dfa491c65d">

Reply:
and this kind of error user can get from any other command where he pass invalid identifier. Only here it was manually handled and converted to different message. I would prefer to keep consistent look everywhere

Reply:
Thank you, looks good to me, let's keep it consistent